msteams-platform/concepts/deploy-and-publish/appsource-publishing-guidance/prepare-app/overview.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Submission checklist 
+description: The checklist to use before publishing your Microsoft Teams app to AppSource
+keywords: teams publish store office publishing checklist
+---
+
+Teams app approval is a free service provided by the Microsoft Teams group that verifies that your app works as described, contains all appropriate metadata, and provides content that would be valuable to an end user.
+
+For your Teams app to be approved, your app must meet the following requirements and guidelines:
+
+* **Distribution Method**: Make sure your app is meant for a store. There are [other options](msteams-platform\concepts\deploy-and-publish\overview.md)
+* **App Functionality**: Your app meets [App Functionality guildeline](msteams-platform\concepts\deploy-and-publish\appsource-publishing-guidance\prepare-app\app-functionality-requirement.md) to deploy your app without submitting to AppSource.
+* **App Detail Page**: Your app meets [App Detail Page Checklist](~/concepts/deploy-and-publish/appsource-publishing-guidance/prepare-app/app-detail-page-checklist.md)
```

Comment:
I don't think "App Detail Page" is the correct name for what this is. It doesn't really describe it well. Maybe "App store listing"?